python/source.py

```diff
@@ -30,26 +30,34 @@ def __init__(self, is_integrated=False):
 
 class ContinuousSource(SourceTime):
 
-    def __init__(self, frequency, start_time=0, end_time=1.0e20, width=0, cutoff=3.0):
+    def __init__(self, frequency=None, start_time=0, end_time=1.0e20, width=0, cutoff=3.0, wavelength=None):
+        if frequency is None and wavelength is None:
+            raise ValueError("Must set either frequency or wavelength in {}.".format(self.__class__.__name__))
+
         super(ContinuousSource, self).__init__()
-        self.frequency = float(frequency)
+        self.frequency = 1 / wavelength if wavelength else frequency
```

Comment:
It doesn't need to be `float(frequency)`?  Hopefully SWIG does the conversion for you if `frequency` is e.g. an integer?

Reply:
Fixed.